src/api-service/__app__/node/__init__.py

```diff
@@ -73,6 +73,8 @@ def delete(req: func.HttpRequest) -> func.HttpResponse:
             context=request.machine_id,
         )
 
+    NodeMessage.clear_messages(node.agent_id)
```

Comment:
Could new messages also arrive in the meantime?

Is there a time later on in the deletion lifecycle where we should clear the messages, instead?

Reply:
moved to Node.delete()